### PR TITLE
[QC-515] Fix analysis json generation in Example and stop polluting src dir

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: Modules/Example/CMakeLists.txt [hunk 1]
```diff
@@ -25,16 +25,10 @@ install(TARGETS QcExample
...
-# the config files being already installed in the right place.
-configure_file("etc/analysisDirect.json" "${CMAKE_INSTALL_PREFIX}/etc/analysisDirect.json")
-configure_file("etc/analysisDerived.json" "${CMAKE_INSTALL_PREFIX}/etc/analysisDerived.json")
-
 # ---- Workflows for analysis ----
 
-o2_add_qc_workflow(WORKFLOW_NAME o2-qc-example-analysis-direct CONFIG_FILE_PATH ${CMAKE_INSTALL_PREFIX}/etc/analysisDirect.json)
-o2_add_qc_workflow(WORKFLOW_NAME o2-qc-example-analysis-derived CONFIG_FILE_PATH ${CMAKE_INSTALL_PREFIX}/etc/analysisDerived.json)
+o2_add_qc_workflow(WORKFLOW_NAME o2-qc-example-analysis-direct CONFIG_FILE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/etc/analysisDirect.json)
+o2_add_qc_workflow(WORKFLOW_NAME o2-qc-example-analysis-derived CONFIG_FILE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/etc/analysisDerived.json)
```
Comment (knopers8) [inline Modules/Example/CMakeLists.txt:31]: The DPL dumps should point to the configuration files in the installation paths, not the ones in source dir.

**Comment by Barthelemy** [comment]
@TimoWilken @ktf It seems that the o2 build failed without producing a log.

File: Modules/Example/CMakeLists.txt (hunk 1 above)
Comment (Barthelemy) [reply Modules/Example/CMakeLists.txt:31]: ah right, that was the thing. It needs to point to the correct path to generate a proper json. Fixing

**Review by davidrohr — APPROVED** [review]
For reference, all problems are fixed for me now with this.

**Pull request merged**